### ISSUE TITLE
Minor text error in /enchant <enchant>

### ISSUE THREAD

**Issue by markusew** [GC-spigot/AdvancedEnchantments#424]
As you can see there's a _Ä_ in the information box before "Description". The config is correct so I assume this must be a minor error in the plugin.
https://i.imgur.com/u49YgsM.png<
This is the config being used
```yaml
        type: ENCHANTED_BOOK
        id: 0
        force-glow: false
        name: "&7Information about &e%enchant% &7enchantment"
        lore:
          - "&eEnchantment &7» &r%ench-colors%"
          - "&eApplies to &7» &r%applies-to%"
          - "&eMax Level &7» &r%max-lvl%"
          - "&eDescription &7» &r%desc%"
```

**Comment by GC-spigot** [comment]
@markusew What plugin version are you using?

**Comment by markusew** [comment]
5.5.3. I see now that this was fixed in 5.5.5. My bad!
